mmv1/third_party/terraform/tests/resource_compute_disk_test.go.erb

```diff
@@ -790,6 +790,7 @@ resource "google_compute_instance_group_manager" "manager" {
   // block on instances being ready so that when they get deleted, we don't try
   // to continue interacting with them in other resources
   wait_for_instances = true
+  wait_for_instances_status = "UPDATED"
```

Comment:
Is this changed due to a failing test, or just to ensure we use it somewhere? If it's just to use it, can we do so in an IGM-specific test?

Reply:
Also, can that test migrate from one to the other?